### PR TITLE
Arch 123 people nodes

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Person content type
- research area content type (for use in person and maybe others)
- publications view fixes

# Needed By (Date)
- soon

# Urgency
- low

# Steps to Test

1. make sure to composer install/update
2. sync to local `blt drupal:sync --site=archaeology --sync-files`
3. ensure the person & research area content types exist
4. use devel generate or manually generate content for research areas, then people, then create some publications and for the author, choose an existing author from the select list
5. view that author's page you choose in step 4
6. ensure the publication you create appears at the bottom and that the layout looks similar to the [mock ups](https://projects.invisionapp.com/share/GMJJYT1NS79#/screens/302442150)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by sherakama** [comment]
Should I wait a little longer for a few more commits to come through or should I start my review?

File: blt/src/Commands/HumsciCommand.php [hunk 1]
```diff
@@ -2,15 +2,25 @@
...
  * Defines commands in the "custom" namespace.
  */
 class HumsciCommand extends AcHooksCommand {
 
+  /**
+   * Log in when drupal:sync finishes.
+   *
+   * @hook post-command drupal:sync
+   */
+  public function postArtifactBuild($result, CommandData $commandData) {
```
Comment (sherakama) [inline blt/src/Commands/HumsciCommand.php]: PostArtifactBuild doesn't sound like the end of drupal:sync. It sounds like what happens after blt:deploy.

**Comment by pookmish** [comment]
@sherakama its all good. i'm branching off this for some views now

File: blt/src/Commands/HumsciCommand.php (hunk 1 above)
Comment (pookmish) [reply blt/src/Commands/HumsciCommand.php]: ooh yeah.. copy paste bad mike

**Comment by sherakama** [comment]
I was just being cheeky. I'll chip away at this review. @boznik too!

**Comment by boznik** [comment]
should i run this blt update?
update_9001001: Adjust Drush 9 Composer contrib directory.

**Comment by pookmish** [comment]
I've never ran the drush command for devel generate. interesting though. 🤔

**Comment by boznik** [comment]
so is this the right mockup? what page is this on?
https://projects.invisionapp.com/share/GMJJYT1NS79#/screens/292457736

**Comment by pookmish** [comment]
woops. i changed the mockup link to https://projects.invisionapp.com/share/GMJJYT1NS79#/screens/302442150

**Comment by boznik** [comment]
FYI devel isn't smart enough to use the auto generated people as authors, so you need to change them manually to get the views to populate

**Comment by boznik** [comment]
FYI, devel genc creates the nodes but uses taxonomy items that don't exist, so you can't edit them without manual work...

**Pull request merged**